Configuration/Geometry/python/GeometrySimDB_cff.py

```diff
@@ -4,12 +4,14 @@
 # Geometry master configuration
 #
 # Ideal geometry, needed for simulation
-from GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff import *
-from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
-from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
-from Geometry.EcalCommonData.ecalSimulationParameters_cff   import *
-from Geometry.HcalCommonData.hcalSimulationParameters_cfi   import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
-from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
-from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
-from Geometry.MuonNumbering.muonGeometryConstants_cff       import *
+
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+
+def _loadGeometryDBDDD(process) :
+    process.load('Configuration.Geometry.GeometryDDDSimDB_cff')
+
+def _loadGeometryDBDD4hep(process) :
+    process.load('Configuration.Geometry.GeometryDD4hepSimDB_cff')
+
+modifyGeometryConfiguration = (~dd4hep).makeProcessModifier(_loadGeometryDBDDD)
+modifyGeometryConfiguration_dd4hep = dd4hep.makeProcessModifier(_loadGeometryDBDD4hep)
```

Comment:
The `ProcessModifier` objects need to have unique labels within CMSSW (technically within a Process). Now there is a conflict with `GeometryRecoDB_cff`.

Reply:
@makortel Thanks. I fixed this.